tests/Resources/ti.network.httpclient.addontest.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.Network.HTTPClient', function () {
+	this.timeout(6e4);
+
+	it('TIMOB-27767 - trigger error callback for invalid URL', function (finish) {
+		var xhr = Ti.Network.createHTTPClient();
+		xhr.setTimeout(6e4);
+
+		xhr.onerror = function (e) {
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.network.httpclient.addontest.js;
  Line: 19;
-->

- :warning: tests/Resources/ti.network.httpclient.addontest.js line 19 – 'e' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)